File: Atividade4.java
import java.util.Scanner;

public class Atividade4 {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        // Variáveis para contagem
        int contadorAcima90kg = 0;

        // Solicitar dados de 7 pessoas
        for (int i = 1; i <= 7; i++) {
            System.out.print("Digite a idade da " + i + "ª pessoa: ");
            int idade = scanner.nextInt();

            System.out.print("Digite o peso da " + i + "ª pessoa: ");
            double peso = scanner.nextDouble();

            // Verificar se o peso é superior a 90 kg e atualizar a contagem
            if (peso > 90) {
                contadorAcima90kg++;
            }
        }

        // Exibir a quantidade de pessoas com mais de 90 kg
        System.out.println("Número de pessoas com mais de 90 kg: " + contadorAcima90kg);

        scanner.close();
    }
}

File: Atividade4.py
def contar_pessoas_acima_90kg():
    # Variáveis para contagem
    contador_acima_90kg = 0

    # Solicitar dados de 7 pessoas
    for i in range(1, 8):
        idade = int(input(f"Digite a idade da {i}ª pessoa: "))
        peso = float(input(f"Digite o peso da {i}ª pessoa: "))

        # Verificar se o peso é superior a 90 kg e atualizar a contagem
        if peso > 90:
            contador_acima_90kg += 1

    # Exibir a quantidade de pessoas com mais de 90 kg
    print(f"Número de pessoas com mais de 90 kg: {contador_acima_90kg}")

# Chamar a função
contar_pessoas_acima_90kg()
